pkg/dto/aerospike_cluster.go

```diff
@@ -170,7 +172,7 @@ type Credentials struct {
 	User *string `yaml:"user,omitempty" json:"user,omitempty" example:"testUser"`
 	// The password for the cluster authentication.
 	Password *string `yaml:"password,omitempty" json:"password,omitempty" example:"testPswd"`
-	// The file path with the password string, will take precedence over the password field.
+	// The file path with the password string.
```

Comment:
Why the comment has been updated?

Reply:
because actually password, password-path and password-key-secret are 3 different ways to read password, they are mutually exclusive (I check it in validation)